x-pack/plugins/security_solution/public/management/pages/policy/store/policy_details/selectors.ts

```diff
@@ -70,16 +73,37 @@ export const policyDetailsForUpdate: (
   }
 });
 
+/**
+ * Checks if data needs to be refreshed
+ */
+export const needsToRefresh = (state: Immutable<PolicyDetailsState>): boolean => {
+  return !state.policyItem && !state.apiError;
```

Comment:
At this point this is super basic but prevents data to be reloaded when switching tabs

Reply:
nitpick: perhaps name this `needsToRefreshPolicyData`?